### ISSUE TITLE
Ignore websocket routes - Swagger schema validation failed on Python 3.7.13

### ISSUE THREAD

**Issue by scott91e1** [pgjones/quart-schema#8]
Hello again,

The schema todo app produces this openapi.json:

```
{"components":{"schemas":{}},"info":{"title":"todo","version":"0.1.0"},"openapi":"3.0.3","paths":{"/":{"post":{"parameters":[],"requestBody":{"content":{"application/json":{"schema":{"properties":{"due":{"format":"date-time","title":"Due","type":"string"},"task":{"title":"Task","type":"string"}},"required":["task","due"],"title":"Todo","type":"object"}}}},"responses":{"201":{"content":{"application/json":{"schema":{"properties":{"due":{"format":"date-time","title":"Due","type":"string"},"task":{"title":"Task","type":"string"}},"required":["task","due"],"title":"Todo","type":"object"}}},"description":"Todo(task: str, due: Union[datetime.datetime, NoneType])"}}}},"/ws":{"get":{"parameters":[],"responses":{}}}},"servers":[],"tags":[]}
```
https://apitools.dev/swagger-parser/online/ produces:

```
Swagger schema validation failed.
  #/paths/~1ws/get/responses must NOT have fewer than 1 properties


Error: Swagger schema validation failed.
  #/paths/~1ws/get/responses must NOT have fewer than 1 properties

    at o (https://apitools.dev/swagger-parser/online/js/bundle.min.js:1:95564)
    at https://apitools.dev/swagger-parser/online/js/bundle.min.js:29:45042

SyntaxError: Swagger schema validation failed.
  #/paths/~1ws/get/responses must NOT have fewer than 1 properties

    at Function.o [as syntax] (https://apitools.dev/swagger-parser/online/js/bundle.min.js:1:95564)
    at validateSchema (https://apitools.dev/swagger-parser/online/js/bundle.min.js:1:6322)
    at SwaggerParser.validate (https://apitools.dev/swagger-parser/online/js/bundle.min.js:1:3391)
```

Any idea as to why?

Regards,

Scott.

**Comment by pgjones** [comment]
I assume the `/ws` path is a websocket route? In which case I think the issue is that there is no way to describe a websocket in OpenAPI.

**Comment by pgjones** [comment]
Fixed in b68eaa0d8b7f1d9a60383ff489e35010f4fd7d6d
